ospec/README.md

```diff
@@ -465,6 +465,14 @@ Asserts that two values are recursively equal
 
 Asserts that two values are not recursively equal
 
+#### Function(String description) o(Function fn).throws(Object constructor)
+
+Asserts that a function throws an Error of the provided constructor
```

Comment:
Suggestion: try revising this to "Asserts that a function throws an instance of the provided constructor", and revising `Object` to `Function`. (Pre-ES6, only functions could be checked, but even now, it's still relatively rare for an object to not both be callable and implement `Symbol.hasInstance`.)